.github/workflows/drivers.yml

```diff
@@ -211,6 +244,39 @@ jobs:
       with:
         junit-name: 'be-tests-mongo-5-0-ee'
 
+  be-tests-mongo-5-0-ssl-ee:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    env:
+      CI: 'true'
+      DRIVERS: mongo
+      MB_MONGO_TEST_USER: metabase
+      MB_MONGO_TEST_PASSWORD: metasample123
+      MB_TEST_MONGO_REQUIRES_SSL: true
+    steps:
+    - uses: actions/checkout@v3
+    - name: Spin up Mongo docker container
+      run: docker run -d -p 27017:27017 --name metamongo metabase/qa-databases:mongo-sample-5.0 mongod --dbpath /data/db2/ --tlsMode requireTLS --tlsCertificateKeyFile /etc/mongo/metamongo.pem --tlsCAFile /etc/mongo/metaca.crt
+    - name: Wait until the port 27017 is ready
+      run: while ! nc -z localhost 27017; do sleep 1; done
+      timeout-minutes: 5
+    - name: Make SSL certificates for Mongo available
```

Comment:
Maybe in a follow up PR, or whenever we re-visit these tests this should say "Make SSl certificates for Mongo available to the host machine". It's not at all obvious this is why we `curl` them even though they are already available in the docker image.